### PR TITLE
Feature/named indices

### PR DESCRIPTION
**Pull request by hkernbach**

### CODE REVIEW

File: src/main/java/com/arangodb/model/FulltextIndexOptions.java [hunk 1]
```diff
@@ -60,6 +61,20 @@ public Integer getMinLength() {
...
+	 * @param name
+	 *            the name of the index
+	 * @return options
+	 */
+	public FulltextIndexOptions name(final String name) {
+		this.name = name;
+		return this;
+	}
+
+	protected String getName() {
```
Comment (mchacki) [inline src/main/java/com/arangodb/model/FulltextIndexOptions.java]: can you please add JavaDoc comments to all getName() implementations.
will not repeat it.

Comment (hkernbach) [reply src/main/java/com/arangodb/model/FulltextIndexOptions.java]: @mchacki all the other "get" methods do not have a "JavaDoc" comment, let's discuss asap.

Comment (mchacki) [reply src/main/java/com/arangodb/model/FulltextIndexOptions.java]: ok ¯\_(ツ)_/¯ java is not my main expertise, If you think this is good than keep it.

Comment (hkernbach) [reply src/main/java/com/arangodb/model/FulltextIndexOptions.java]: Will put it on my list and discuss with Michele.

**Pull request merged**